startup.S

```diff
@@ -4,6 +4,7 @@
 /* Exception vector. */
 .section .exception
 .globl __reset_vector
+.org 0x0
```

Comment:
I don't understand what [.org](https://sourceware.org/binutils/docs/as/Org.html) does here. Can you explain?